substrate-node/pallets/test_offchain_worker_setup.py

```diff
@@ -0,0 +1,19 @@
+from re import sub
```

Comment:
Same curiosity as before... and it looks very similar to first part of previous python file?
Do we really need to repeat code?

Reply:
Oh we are supposed to add Bob's keys here. And indeed it is very similar code but as this script will be removed or reworked into something else I guess it's ok to keep it as it is.